src/app/course/_test/mock.ts

```diff
@@ -302,4 +323,7 @@ export const MOCK_UQJ2: UQJ = {
     variables: JSON.parse('{}'),
     variables_errors: [],
     is_checkbox: false,
+    report: {} as QuestionReport,
```

Comment:
you shouldn't need to cast it you can just make the value undefined, or just omit the value entirely. The absence of an object should indicate deletion

Reply:
I made the value undefined!